.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.205.2/containers/typescript-node/.devcontainer/base.Dockerfile
+
+# [Choice] Node.js version (use -bullseye variants on local arm64/Apple Silicon): 16, 14, 12, 16-bullseye, 14-bullseye, 12-bullseye, 16-buster, 14-buster, 12-buster
+ARG VARIANT="16-bullseye"
```

Comment:
should be: "ARG VARIANT"   , will then be set from devcontainer.json

Reply:
I'm not following

Reply:
Sorry, I read again the docker docu. It is correct. It will be overwritten by whatever is in devcontainer.json